src/qt/sendcoinsdialog.cpp

```diff
@@ -369,6 +369,8 @@ void SendCoinsDialog::on_sendButton_clicked()
         accept();
         CoinControlDialog::coinControl()->UnSelectAll();
         coinControlUpdateLabels();
+        // Navigate to transaction history page:
```

Comment:
Coin control fields, custom change address and selected inputs, are already cleared after you send.

Reply:
Please verify custom change address checkbox and line edit because here they are not cleared.

Reply:
You're right. Turns out #12432 fixes that, because  `SendCoinsDialog::accept()` is called when the transaction is sent, which in turn calls `clear()`.